fenix/app/src/test/java/org/mozilla/fenix/components/toolbar/DefaultBrowserToolbarMenuControllerTest.kt

```diff
@@ -823,6 +823,24 @@ class DefaultBrowserToolbarMenuControllerTest {
         verify { navController.navigate(turnOnSyncDirections, null) }
     }
 
+    @Test
```

Comment:
Could you also add an additional test entry at [`verifyPageThreeDotMainMenuItems`](https://searchfox.org/mozilla-mobile/rev/b4fff7f0b3d73d23a8baf7711dc8e0163518d2a1/firefox-android/fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/ThreeDotMenuMainRobot.kt#90) for the translate menu item set behind the translations enabled flag?

 (I think that test might start failing when translations aren't behind a flag, so having the test ready to go will be helpful.)

Reply:
Done